### PR TITLE
HBSD: hardenedbsd/secadm: split off -kmod port

### PR DESCRIPTION
**Pull request by xmj**
* make secadm master port
* create secadm-kmod slave port
* hook secadm-kmod into the build
* usecase: install secadm into jails and
  secadm-kmod on jailhosts.

### CODE REVIEW

**Comment by xmj** [comment]
I should probably have done the branching differently. Please merge https://github.com/HardenedBSD/hardenedbsd-ports/pull/9 before proceeding.

**Comment by xmj** [comment]
Requires https://github.com/HardenedBSD/secadm/pull/29/ to be merged in order to work properly.

File: hardenedbsd/secadm-kmod/pkg-plist [hunk 1]
```diff
@@ -0,0 +1 @@
+/%%KMODDIR%%/secadm.ko
```
Comment (Sp1l) [inline hardenedbsd/secadm-kmod/pkg-plist]: Merge this with the `Makefile`

> There is only one case when pkg-plist can be omitted from a port. If the port installs just a handful of files, list them in PLIST_FILES, within the port's Makefile.

File: hardenedbsd/secadm-kmod/Makefile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+COMMENT=	HardenedBSD Security Administration - Kernel module
+
+USES=		kmod
+KMOD=		yes
+
+USE_GITHUB=	yes
+pre-build:
+	(cd ${WRKSRC}; make depend)
+
+PLIST=	${.CURDIR}/pkg-plist
```
Comment (Sp1l) [inline hardenedbsd/secadm-kmod/Makefile]: `PLIST_FILES= /${KMODDIR}/secadm.ko`

Or does this add the files from master as well? (consider changing master to PLIST_FILES as well, 6 is a big handful :smile:)

Comment (xmj) [inline hardenedbsd/secadm-kmod/Makefile]: Unless the PLIST is explicitly overridden ISTR that it'd pull in all files from master PLIST, and fail to package.

**Pull request merged**